week-5/A-dom-manipulation/exercise.js

```diff
@@ -4,30 +4,46 @@ Task 1
 
 Write JavaScript below that changes the background colour of the page when the 'Change colour' button is clicked.
 */
-
-
+var updateTitleBtn = document.querySelector("#bgrChangeBtn");
+updateTitleBtn.addEventListener("click", function() {
+  var myElement = document.querySelector("#bgrChangeBtn");
+  myElement.style.backgroundColor = "red";
+});
 
 /*
 Task 2
 ======
-
 When a user clicks the 'ALERT' button, an alert box should pop up with the text "Thanks for visiting Bikes for Refugees!"
 */
-
-
-
+var updateTitleBtn = document.querySelector("#alertBtn");
+updateTitleBtn.addEventListener("click", function() {
+  alert("Thanks for visiting Bikes for Refugees!");
+});
 /*
 Task 3
 ======
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
-
-
-
+var updateTitleBtn = document.querySelector("#addTextBtn");
+updateTitleBtn.addEventListener("click", function() {
+  var myElement = document.querySelector(".buttons");
+  var myParagraph = document.createElement("p");
+  document.querySelector("#mainArticles").appendChild(myParagraph);
+  myParagraph.innerText = "Read more below.";
```

Comment:
I think this does work, but it does the operations in a strange order. We want to create the `myParagraph` element with all of it's text, then append it into the DOM.

So I would swap lines 32 and 33.

Reply:
yes i did